src/server/api.js

```diff
@@ -63,16 +63,42 @@ function submitApplication(req, res) {
       if (err) {
         debug(`voaService response had error ${err}`);
         // TODO(awong): This may leak server config info on error. Is that a problem?
-        res.status(500).send({ error: err });
+        res.status(500).json({ error: err });
```

Comment:
I tinkered with changing between `.json()` and `.send()` using `curl` as a client and the Chrome Boomerang app, and there was no observable difference. Both returned identical strings from JSON objects and both had _Content-Type: application/json_ in the returned headers.